### PR TITLE
update documentation

### PR DESCRIPTION
**Pull request by arodrime**
Removing the documentation from the Readme as it now lives in the wiki.
Update gitter --> ASF Slack channel #cassandra-medusa

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,575 +14,6 @@
 # limitations under the License.
 -->
 
-[![Join the chat at https://gitter.im/thelastpickle/cassandra-medusa](https://badges.gitter.im/thelastpickle/cassandra-medusa.svg)](https://gitter.im/thelastpickle/cassandra-medusa?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+For any support questions or discussion you'd like to start or contribute to, join the channel #cassandra-medusa on ASF slack at [http://s.apache.org/slack-invite](http://s.apache.org/slack-invite)
```
Comment (adejanovski) [inline README.md:17]: I'd go with:

For user questions and general/dev discussions, please join the #cassandra-medusa channel on the ASF slack at [http://s.apache.org/slack-invite](http://s.apache.org/slack-invite)

File: README.md [hunk 2]
```diff
@@ -14,575 +14,6 @@
...
-[2019-09-04 12:56:19] INFO: - Name: 2019090407
-[2019-09-04 12:56:19] INFO: - Finished: True
-[2019-09-04 12:56:19] INFO: - Details - Node counts
-[2019-09-04 12:56:19] INFO: - Complete backup: 180 nodes have completed the backup
-[2019-09-04 12:58:47] INFO: - Total size: 94.69 TiB
-[2019-09-04 12:58:55] INFO: - Total files: 5168096
-```
-
-When used with `--push-metrics`, Medusa will push completion metrics to the configured monitoring system.
+To get started with Medusa, backups and Restore for Cassandra, use the documentation in the [wiki of the project](../../wiki)
```
Comment (adejanovski) [inline README.md:19]: > with Medusa, backups and Restore for Cassandra,

"backups" should have an uppercase B to be consistent with the rest of the text.

Comment (adejanovski) [reply README.md:19]: I'd also trade `use` with `check`

**Comment by arodrime** [comment]
Thanks Alex. It's merged.

**Pull request merged**